src/components/UpgradeTableOfContents.tsx

```diff
@@ -1,47 +1,10 @@
 import React from "react"
-import { motion } from "framer-motion"
-import { Link } from "gatsby"
-import styled from "@emotion/styled"
+import { Box, Link, ListItem, UnorderedList } from "@chakra-ui/react"
 
 import type { Item as ItemTableOfContents } from "./TableOfContents"
 
 const customIdRegEx = /^.+(\s*\{#([A-Za-z0-9\-_]+?)\}\s*)$/
 
-const Aside = styled.aside`
-  padding: 0rem;
-  text-align: right;
-  margin-bottom: 2rem;
-  overflow-y: auto;
-`
-
-const OuterList = styled(motion.ul)`
```

Comment:
Motion ul wasn't doing anything? I've removed it.

Reply:
🤔 yeah, I don't see that we are doing any animation with it. Fine with removing it.